src/common/UnkAngleFunc1.inc.c

```diff
@@ -0,0 +1,16 @@
+#include "common.h"
+
+s32 N(UnkAngleFunc1)(f32 angle) {
+    s32 angleDiff;
+
+    if (angle >= 270.0f) {
+        angleDiff = 360.0f - angle;
+    } else if (angle >= 180.0f) {
+        angleDiff = angle - 180.0f;
+    } else if (angle >= 90.0f) {
+        angleDiff = 180.0f - angle;
+    } else {
+        angleDiff = angle;
+    }
+    return 0xFF - angleDiff - angleDiff;
```

Comment:
This `0xFF` is weird, could it perhaps be `-1` (with this func returning `s8`)?

Reply:
unfortunately no, but ther's probably something else I'm missing